app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] The association should be `belongs_to :author, class_name: 'User'`

Reply:
* done 👍